tests/Command/RunCommandTest.php

```diff
@@ -1197,23 +1197,14 @@ public function testExecuteSuccessWithUsageOfSlicer(array $configDataOuputStorag
             }
         }
 
-        $configDataStorage = [
-            'input' => [
-                'tables' => [
-                    [
-                        'source' => $tableId,
-                        'destination' => 'source.csv',
-                    ],
-                ],
-            ],
-        ];
-
+        $configDataStorage = [];
         if ($configDataOuputStorage) {
             $configDataStorage['output'] = $configDataOuputStorage;
         }
 
         $jobData = [
             'componentId' => 'keboola.python-transformation',
+            'tag' => 'latest',
```

Comment:
to tady k něčemu je?

Reply:
Fuj, to se tam dostalo omylem.
Mam upraveny docker bundle aby nepulloval image vzdy + latest aby se to nestanilo stahovat veci co nemam pulnute lokalne.
Jdu to vyhodit.

Reply:
vyhodil jsem to